.gitignore

```diff
@@ -3,6 +3,10 @@
 # C extensions
 *.so
 
+# vim temporal files
```

Comment:
Sorry Adam, and thank you for your patience. I am learning quite a lot of things today..